enhancements/two-node-fencing/tnf.md

```diff
@@ -565,6 +565,14 @@ possible, but it is not sufficient for the operation of a healthy cluster longte
 
 Because of the flexibility it can offer users to adjust their cluster capacity according to their needs, transitioning between control-plane topologies may be introduced for evaluation in a future enhancement.
 
+#### Regarding Compute Nodes
+
+As currently envisioned, it doesn't make sense for a customer to install a new TNF cluster with compute nodes. If the cluster has compute capacity for 3 nodes, it should always be installed as a highly available (3 control-plane) cluster. This beomes more complicated for customers that discover after installing a fleet of TNF clusters that they need to add compute compacity to meet a rise in demand. The preferred solution for this would be to support some kind of topology transition to a highly available compact (3-node) cluster. This is not currently supported, and may never be in OpenShift.
+
+The alternative that we expect to be asked for is to support compute nodes in TNF. For this reason, we'd like to avoid investing heavily in preventing compute nodes from joining the cluster - at install or runtime. It's worth noting that no variant of OpenShift currently actively prevents compute nodes from being added to the cluster.
+
+The only boundary we'd set to declare compute nodes as unsupported in documentation.
```

Comment:
My apologizes if my question was already discussed :-)
But this statement seems a bit counter intuitive to general rule of thumb in which we try to make sure that we prevent users from taking unsupported actions.
I'm wondering as to the reason for this exception :thinking: